.github/workflows/build_library.yml

```diff
@@ -47,10 +47,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - macos-version: "macos-10.15"
```

Comment:
No, we don't care about more recent versions. Initially, we only stuck with the old workers because they used to be available faster. 

Rust MacOS builds are backwards compatible usually, so we can run everything with the default IIRC